misc/extension_api_validation/4.5-stable.expected

```diff
@@ -128,3 +128,11 @@ Validate extension JSON: Error: Field 'builtin_classes/PackedVector3Array/method
 Validate extension JSON: Error: Field 'builtin_classes/PackedVector4Array/methods/bsearch': is_const changed value in new API, from false to true.
 
 bsearch method made const. Compatibility methods registered.
+
+
+GH-112308
+---------
+Validate extension JSON: Error: Field 'classes/Animation/methods/get_length/return_value': meta changed value in new API, from "float" to "double".
+Validate extension JSON: Error: Field 'classes/Animation/methods/set_length/arguments/0': meta changed value in new API, from "float" to "double".
+
+Return type and parameter changed from real_t to double to match internals. No compatibility methods registered because it breaks C# builds.
```

Comment:
> No compatibility methods registered because it breaks C# builds.

What do you mean by this? That since there's a compile-time error, there's no risk of breaking at runtime?

If yes, that misses the point: this is primarily about GDExtension (as indicated by file path), although it covers C# as well. GDExtensions must remain _binary compatible_, i.e. an extension compiled against Godot 4.1 must still run without UB under Godot 4.5.

In this case that is true because `float` and `double` are represented the same in GDExtension's FFI (namely as `double`), but that means you should change the reasoning here.